docs/UserGuide.md

```diff
@@ -151,6 +145,18 @@ Examples:
 * `find John Kite` returns a contact with name `john kite` and another with description `Kite manufacturer`.
 * `find Sun` returns a contact with name `Joe Sun` and another with description `Works at Sun Tools`.
 
+#### Listing all contacts: `list`
+
+Shows a list of all contacts in your contact list.
+
+Format: `list`
+
+<div markdown="span" class="alert alert-primary">:bulb:
+
+**Tip:** Use this command when you want to list all your contacts back after using the `find` command. 
+Refer to [Finding contacts](#finding-contacts-find) above for details of the `find` command. 
```

Comment:
Would hyperlinking just once in the first `find` code in line 156 and deleting the entire line 157 be better for succinctness? Here and elsewhere.

Reply:
Didn't amend this cos I feel the hyperlink and `find` formatting doesn't really look well together.